lib/imagec/docker.go

```diff
@@ -310,12 +315,16 @@ func FetchImageBlob(ctx context.Context, options Options, image *ImageWithMeta,
 }
 
 // FetchImageManifest fetches the image manifest file
-func FetchImageManifest(ctx context.Context, options Options, progressOutput progress.Output) (*Manifest, error) {
+func FetchImageManifest(ctx context.Context, options Options, schemaVersion int, progressOutput progress.Output) (interface{}, string, error) {
 	defer trace.End(trace.Begin(options.Image + "/" + options.Tag))
 
+	if schemaVersion != 1 && schemaVersion != 2 {
+		return nil, "", fmt.Errorf("Unknown schema version %d requested!", schemaVersion)
```

Comment:
well, then we break golint because we capitalize the first letter everywhere in our code base.